apps/meteor/client/sidebar/sections/OmnichannelSection.tsx

```diff
@@ -41,7 +41,12 @@ const OmnichannelSection = (props: typeof Box): ReactElement => {
 				{isCallEnabled && <OmnichannelCallToggle />}
 				<OmnichannelLivechatToggle />
 				{hasPermissionToSeeContactCenter && (
-					<Sidebar.TopBar.Action data-tooltip={t('Contact_Center')} icon='address-book' onClick={(): void => handleRoute('directory')} />
+					<Sidebar.TopBar.Action
+						data-tooltip={t('Contact_Center')}
+						aria-label={t('Contact_Center')}
```

Comment:
the prop `title`  is not enough for this case?

Reply:
@ggazzo We had problems before on a few browsers where both, the native tooltip and custom tooltip were displayed at the same time when using `title`. That's why we're using `data-tooltip` here. The `aria-label` helps with the accessibility that was missing.